drivers/overlay/overlay.go

```diff
@@ -2546,6 +2546,20 @@ func nameWithSuffix(name string, number int) string {
 	return fmt.Sprintf("%s%d", name, number)
 }
 
+func checkIfAllNesseseryFilesExistsInLayerStore(target string) error {
```

Comment:
- Typo in “necessary”
- The graph driver is all about layers; to be specific for this feature, “in additional layer store”.
- (absolutely non-blocking: consider naming this in a way that shows the relationship to `getAdditionalLayerPath`; maybe `validateOneAdditionalLayerPath` or something like that.)